README.txt

```diff
@@ -40,6 +41,25 @@ ASN.1 definitions. The command to do this is::
 It will print the ``pyasn1`` equivalent of ``source.asn1`` to stdout.
 
 
+Usage with Quick DER
+--------------------
+
+Quick DER is a library found on https://github.com/vanrein/quick-der
+A separate generator exists in ``asn1ate`` to create the parser bytecode
+and overlay structures for this format.  The command to do this is::
+
+  $ python .../asn1ate/asn2quickder.py source.asn1
+
+It will store an include file suitable for use with ``Quick DER`` in ``source.h``,
+which is the source file name with its extension changed to ``.h``.
+
+This tool can be used for your private ASN.1 projects, but one purpose of
+Quick DER is also to have a developer's toolkit with precompiled header
```

Comment:
"precompiled" in what sense? You're not talking about "precompiled headers" in the sense MSVC and Clang are using it, right?